kinto/core/storage/memory.py

```diff
@@ -336,6 +342,8 @@ def apply_filters(records, filters):
         COMPARISON.IN: operator.contains,
         COMPARISON.EXCLUDE: lambda x, y: not operator.contains(x, y),
         COMPARISON.LIKE: lambda x, y: re.search(y, x, re.IGNORECASE),
+        COMPARISON.CONTAINS_ALL: contains_all_filtering,
+        COMPARISON.CONTAINS: lambda record_value, search_term: set(record_value[1]).intersection(set(search_term[1]))
```

Comment:
I'd like to understand where this [1] comes from it is a bit unclear to me right now.